tfhe/src/shortint/server_key/tests/mod.rs

```diff
@@ -1,3 +1,4 @@
 pub mod noise_level;
-pub mod shortint;
+pub mod parametrized_test;
+pub mod parametrized_test_bivariate_pbs_compliant;
```

Comment:
the split does not make sense I believe as we discussed ?

Reply:
Even if eventually, we'd like to have many more files, I think, it's better to have two files rather than one

Reply:
yes but the split does not make sense as some tests are not bivar PBS compliant in the corresponding file :/